xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.cpp

```diff
@@ -1055,10 +1055,6 @@ int CRenderManager::WaitForBuffer(volatile std::atomic_bool&bStop, int timeout)
     m_presentevent.wait(lock, std::min(50, timeout));
     if (endtime.IsTimePast() || bStop)
     {
-      if (timeout != 0 && !bStop)
-      {
-        CLog::Log(LOGWARNING, "CRenderManager::WaitForBuffer - timeout waiting for buffer");
-      }
```

Comment:
I wonder if this could be kept. Now when the other side (e.g. the player user) says OUTPUT_AGAIN, it should receive something or better present the last buffer once more. (Why keeping: if we are not paused, but really render is stuck, then we would not see it anymore. But I admit I don't have a better idea at hand.

Btw. do you still see a 100% CPU usage when this block is hit?

Reply:
If we keep that log here, it will keep spamming. I guess introducing a new parameter for this would be a hack?
As for cpu usage: I see no real difference between play and pause state. Both are ~50% CPU

Reply:
The key thing here is that the RenderManager should have no idea about player's internal state. Changing the interface of rendermanager to introduce something like: Pause() though would work.

Never the less out of scope of this PR. We could make a it a LOGDEBUG, then at least if someone wants to debug these things they'd see it and normal users won't?